example/package.json

```diff
@@ -3,25 +3,26 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "clear": "node ./node_modules/react-native/local-cli/cli.js start --reset-cache",
+    "clear": "react-native start --reset-cache",
     "install-pods": "cd ios && pod install && cd ..",
-    "start": "node ./node_modules/react-native/local-cli/cli.js start",
+    "start": "react-native start",
     "test": "jest",
     "flow": "flow"
   },
   "dependencies": {
-    "react": "16.8.6",
-    "react-native": "0.60.3"
+    "react": "16.9.0",
```

Comment:
it's 61 is in rc1 now. but I don't care much. I must admit I actually hate the static-example pattern in that it's a treadmill to keep things in sync and keep up. That's why rn-diff-purge (under the hood of upgrade-helper) beat rn-diff. This will be out of date in a week. I'd prefer something like this https://github.com/react-native-community/react-native-device-info/blob/master/refresh-example.sh - in the absence of that, sure.